eks/cluster-loader/clusterloader2/cluster-loader.gotmpl

```diff
@@ -42,21 +42,34 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: clusterloader2
-  namespace: clusterloader2
+  namespace: cl2
 spec:
   template:
     spec:
       containers:
-      - name: clusterloader2
-        image: {{ .Image }}
-        command: ["/clusterloader"]
-        args: {{ .TestArgs }}
-        volumeMounts:
-        - name: config
-          mountPath: /etc/config
+        - name: clusterloader2
+          image: {{ .Image }}
+          command: ["/clusterloader"]
+          args: {{ .TestArgs }}
+          volumeMounts:
+            - name: config
+              mountPath: /etc/config
+            - name: reports
+              mountPath: /usr/share
+              readOnly: false
+        - name: results
+          image: amazon/aws-cli:latest
+          command: ["/bin/bash"]
+          args: {{ .S3Command }}
```

Comment:
I might imbed the entire script you've created into the go template. Right now, it's difficult to understand what's going on in the golang code with a cursory read. 

The only variable you have is .S3URI, so you could just substitute that in the gotemplate.

Reply:
As per Jacob's comment at the top, will do.